### ISSUE TITLE
Code Table Request - new part attribute: preparation method

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#6802]
### Goal
_Describe what you're trying to accomplish. This is the only necessary step to start this process. The Committee is available to assist with all other steps. Please clearly indicate any uncertainty or desired guidance if you proceed beyond this step._

Kits are available for DNA extraction. Rather than put this as free text in a part preservation attribute method field, we would like to be able to use a code table to pick standard methods from. 

### Context
_Describe why this new value is necessary and existing values are not._

The current OGL data includes extraction method with a specific list of terms

QIAcube
BioSprint96
Qiagen DNeasy Blood and Tissue Kit
NEB Monarch Genomic DNA Purification Kit
Phenol-chloroform
Ethanol precipitation
Whole genome amplification
Other
Unknown

### Table
_Code Tables are http://arctos.database.museum/info/ctDocumentation.cfm. Link to the specific table or value. This may involve multiple tables and will control datatype for Attributes. OtherID requests require BaseURL (and example) or explanation. Please ask for assistance if unsure._

https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecpart_attribute_type

### Proposed Value
_Proposed new value. This should be clear and compatible with similar values in the relevant table and across Arctos._

preparation method

### Proposed Definition
_Clear, complete, non-collection-type-specific **functional** definition of the value. Avoid discipline-specific terminology if possible, include parenthetically if unavoidable._

A standard method used in preparation of the part. Details should be provided in the attribute method.

### Collection type
_Some code tables contain collection-type-specific values. ``collection_cde`` may be found from https://arctos.database.museum/home.cfm_

N/A

#### Attribute data type
_If the request is for an attribute, what values will be allowed?
free-text, categorical, or number+units depending upon the attribute (TBA)_

categorical

#### Attribute controlled values
_If the values are categorical (to be controlled by a code table), add a link to the appropriate code table. If a new table or set of values is needed, please elaborate._

A new code table will be needed. suggest ctpreparation_method

Values that will be immediately proposed:

term | description 
-- | -- 
QIAcube | DNA is extracted using a Qiagen QIAcube
BioSprint96 | DNA is extracted using a Qiagen BioSprint96
Qiagen DNeasy Blood and Tissue Kit | DNA is extracted using a Qiagen DNeasy Blood and Tissue Kit
NEB Monarch Genomic DNA Purification Kit | DNA is extracted using a New England Biolabs Monarch Genomic DNA Purification Kit
phenol-chloroform |  DNA is extracted using a phenol-chloroform extraction (not kit-based)
ethanol precipitation |  DNA is extracted using an ethanol precipitation (not kit-based)
whole genome amplification |   
other | DNA is extracted by an other known method that is specified in methods
unknown | DNA is extracted by an unknown method 

#### Attribute units
_if numerical values should be accompanied by units, provide a link to the appropriate units table._

N/A

#### Part preservation attribute affect on "tissueness"
_if a new part preservation is requested, please add the affect it would have on "tissueness": No Influence, Allows, or Denies_

Do we need something here? Nope - https://github.com/ArctosDB/data-migration/issues/1816#issuecomment-1747730016

### Priority
_Please describe the urgency and/or choose a priority-label to the right. You should expect a response within two working days, and may utilize [Arctos Contacts](https://arctosdb.org/contacts/) if you feel response is lacking._

### Example Data
_Requests with clarifying sample data are generally much easier to understand and prioritize. Please attach or link to any representative data, in any form or format, which might help clarify the request._

### Available for Public View
_Most data are by default publicly available. Describe any necessary access restrictions._

### Helpful Actions

- [x] Add the issue to the [Code Table Management Project](https://github.com/ArctosDB/arctos/projects/13#card-31628184).

- [x] Please reach out to anyone who might be affected by this change. Leave a comment or add this to the Committee agenda if you believe more focused conversation is necessary.

@ArctosDB/arctos-code-table-administrators @happiah-madson @falco-rk 

## Approval
_All of the following must be checked before this may proceed._

_The [How-To Document](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html) should be followed. Pay particular attention to terminology (with emphasis on consistency) and documentation (with emphasis on functionality). **No person should act in multiple roles**; the submitter cannot also serve as a Code Table Administrator, for example._

- [x] Code Table Administrator[1] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [x] Code Table Administrator[2] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [x] DBA - The request is functionally acceptable. The term is not a functional duplicate, and is compatible with existing data and code.
- [x] DBA - Appropriate code or handlers are in place as necessary. (ID_References, Media Relationships, Encumbrances, etc. require particular attention)

## Rejection
_If you believe this request should not proceed, explain why here. Suggest any changes that would make the change acceptable, alternate (usually existing) paths to the same goals, etc._

1. _Can a suitable solution be found here? If not, proceed to (2)_
2. _Can a suitable solution be found by Code Table Committee discussion? If not, proceed to (3)_
3. _Take the discussion to a monthly Arctos Working Group meeting for final resolution._

## Implementation
_Once all of the Approval Checklist is appropriately checked and there are no Rejection comments, or in special circumstances by decree of the Arctos Working Group, the change may be made._

- [ ] Review everything one last time.  Ensure the How-To has been followed. Ensure all checks have been made by appropriate personnel.

- [ ] Add or revise the code table term/definition as described above. Ensure the URL of this Issue is included in the definition. URLs should be included *_as text_*, separated by spaced pipes. Do not include HTML in definitions.

_Close this Issue._

_**DO NOT** modify Arctos Authorities in any way before all points in this Issue have been fully addressed; data loss may result._

## Special Exemptions

_In very specific cases and by prior approval of The Committee, the approval process may be skipped, and implementation requirements may be slightly altered. Please note here if you are proceeding under one of these use cases._

1. _Adding an existing term to additional collection types may proceed immediately and without discussion, but doing so may also subject users to future cleanup efforts. If time allows, please review the term and definition as part of this step._
2. _The Committee may grant special access on particular tables to particular users. This should be exercised with great caution only after several smooth test cases, and generally limited to "taxonomy-like" data such as International Commission on Stratigraphy terminology._

**Comment by mkoo** [comment]
This seems straightforward.
Why isnt there a description for 'whole genome amplification'? (there are kits for that too but that could be in remarks ofc)
how about:
"non‐selective amplification of the whole genome sequence" or something generic since it's the only one without a description

meanwhile, I am marking this as otherwise ok by me

**Comment by dustymc** [comment]
>description for 'whole genome amplification'?

It's just an example of a possible value - it'll get its own request. This is just a request for the attribute type and code table (which will eventually hold those values, or whatever falls out of the request process).

**Comment by mkoo** [comment]
> > description for 'whole genome amplification'?
> 
> It's just an example of a possible value - it'll get its own request. This is just a request for the attribute type and code table (which will eventually hold those values, or whatever falls out of the request process).

ah, ofc then goferit!

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
This seems fine to me.

**Comment by Jegelewicz** [comment]
@dustymc this is awaiting -  Appropriate code or handlers are in place as necessary. (ID_References, Media Relationships, Encumbrances, etc. require particular attention)

Any problems to overcome?

**Comment by dustymc** [comment]
>this is awaiting 

Yes.

<img width="579" alt="Screenshot 2023-10-09 at 08 05 34" src="https://github.com/ArctosDB/arctos/assets/5720791/dff6ad5f-4ea8-4945-855a-0824847215e6">

>problems

Just getting through the maybe-more-pressing Issues that pop into my inbox....

**Comment by Jegelewicz** [comment]
I need to request the values as listed...

**Comment by happiah-madson** [comment]
Just checking in on this, I have some students with piles of DNA extraction data that they need to enter before the end of the month!

**Comment by Jegelewicz** [comment]
@happiah-madson the term is created, but not the values - see all those other issues...

**Comment by happiah-madson** [comment]
> @happiah-madson the term is created, but not the values - see all those other issues...

ope, somehow i missed it in the part attribute table, but I see it now!

**Comment by dustymc** [comment]
I'm reopening this in light of https://github.com/ArctosDB/code-table-work/issues/26#issuecomment-2338598288

The definition above and the data in https://arctos.database.museum/info/ctDocumentation.cfm?table=ctpreparation_method make me think that IS NOT a preservation.

Can we shore up the definition to clearly separate preparation method from preservation?

If not, this attribute should probably be merged into preservation - we should not have arbitrary categories (and preservation is a sort of catch-all anyway).

Here's who's used preparation method

```

 guid_prefix | count 
-------------+-------
 ANSP:Host   |  4460
 ASNHC:Mamm  |     7
 MSB:Arth    |     1
 OGL:Genomic | 34519
 UTEP:Bird   |     1
 UTEP:ES     |     1
 UWBM:PB     |  7014
 UWBM:PR     | 14574
```


 @campmlc
 @WaigePilson
 @mvzhuang
 @rwilhoyt
 @happiah-madson
 @falco-rk


HELP!!

**Comment by happiah-madson** [comment]
Yo, the way we use preparation method is 1000000000% not preservation. Our fields are the specific _methods_ that are used to generate a DNA sample. Do not make us use preservation, please, for the love of all things holy.

**Comment by WaigePilson** [comment]
> Can we shore up the definition to clearly separate preparation method from preservation?

I would suggest (slight modifications from what is already published for these code tables)--
Preparation method: Preparation method describes <s>an intentional action<s> a process, protocol, or action that alters the object for a particular purpose, which may include a published or commercially available routine and workflow.
Preservation: Controlled vocabulary describing how a part is physically preserved or stored in the collection. Terms require a status for "Tissueness", which indicates whether the part is eligible for genetic sequencing or not.

For example: I added a preparation method of "processed for phytoliths" which I use to document when a sample of leaf tissue, sediment, soil, rock, etc. has been processed for phytolith content (a multi-step, multi-day lab protocol well documented in literature in my field). I use a part preservation of "slide" for the resultant 1-many slides that are produced after that protocol, or "desiccated" for the vials of dried phytolith residue resulting from that protocol.

Regarding https://github.com/ArctosDB/code-table-work/issues/26 I'd think this meets the definition of a "preparation method" but I am not particularly familiar with these protocols.

And yes, many many many agreements with @happiah-madson

**Comment by happiah-madson** [comment]
Jumping onto what @WaigePilson said, we do a similar thing:

preparation method = DNA extraction method
preservation = the liquid the extracted DNA is eluted in. (I actually think this is a little iffy, but it is the best option that we currently have).

**Comment by happiah-madson** [comment]
Also, if there is more context about what the subculturing field would contain, i'm happy to weigh in? Is it the fact that it has or has not been subcultured (and is that the same as tracking passages?). I know I'm jumping in with @campmlc on Wednesday, so if it's too much to catch me up, no worries.

**Comment by dustymc** [comment]
> more context about what the subculturing field would contain

The proposal is for part **preservation** to get a new value 'subculturing' defined as 'The act of ...' - see https://github.com/ArctosDB/code-table-work/issues/26

My current understanding is that 'subculturing' as an act belongs in **preparation** method, not preservation. I think I'm hearing that I'm not completely lost on that....

**Comment by dustymc** [comment]
Thanks all, I think we have this right, added link to this to the docs spreadsheet, closing.

@WaigePilson I think maybe the 'intentional' in there is to differentiate this from condition:

* ethanol precipitation==intentional==this
* 'left in hot car for a week'==oops==condition


Even though both change the nature of the material and what can be done with it.

**Comment by WaigePilson** [comment]
> @WaigePilson I think maybe the 'intentional' in there is to differentiate this from condition:

Ah, gotcha. I think we could elaborate on intentional action to say "intentional process, protocol, or action" maybe. But otherwise I would say the definitions on the code table pages accurately describe what I think these terms mean and how they are different.
